Automattic/wp-calypso
# Performance Profiler: animate loading screen tips

Closes https://github.com/Automattic/dotcom-forge/issues/9108

## Proposed Changes

* Animate loading screen tips

## Why are these changes being made?


https://github.com/Automattic/dotcom-forge/issues/9108

## Testing Instructions



* Go to `/speed-test` and start a new test
* Check that the tips on the loading screen are animated when replaced

https://github.com/user-attachments/assets/afb87d8a-fe95-4c3e-9812-8ecfe3fc84c6

## Pre-merge Checklist
